decidim-participatory_processes/app/views/decidim/participatory_processes/participatory_processes/show.html.erb

```diff
@@ -48,6 +48,8 @@ edit_link(
           </div>
         </div>
       <% end %>
+
+      <%= assemblies_for_participatory_process(participatory_process_assemblies) if participatory_process_assemblies.present? %>
```

Comment:
Would it be possible to change the way you introduce related assemblies?
I mean, instead of a partial template, to use a card_for just like it's being done above for related processes?
In this way, we avoid having hardcoded the related assembly's template and its classes, in order that if a possible redesign of the cards is carried out later, this will also be affected without any problem.

![image](https://user-images.githubusercontent.com/9702463/79242055-1ca36d80-7e74-11ea-82f1-c93fc4dc5aab.png)